Library/Homebrew/formula_creator.rb

```diff
@@ -84,7 +84,7 @@ def write_formula!
           r.owner = self
           filepath = r.fetch
           html_doctype_prefix = "<!doctype html"
-          if File.read(filepath, html_doctype_prefix.length).downcase.start_with?(html_doctype_prefix)
+          if File.read(filepath, 100).strip.downcase.start_with?(html_doctype_prefix)
```

Comment:
Please add a comment, and ideally turn this into a constant/variable, explaining why 100 is used otherwise it seems arbitrary.

Reply:
@MikeMcQuaid the number is arbitrary, but I've added the comment that explains the logic. Please take a look.